### DISCUSSION TITLE
the say op only seems to play one voice

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#334] [! Welcome - readme first!]
The say op only seems to speak in one voice.

**Comment by marioepsley** [comment]
would also be cool if we could get the audio out, to effect and or analyse

**Reply by steam0r** [reply]
this is not really possible as the browser-api doesn't give an audiostream, i fixed the voice-selection on dev.cables.gl though

**Reply by marioepsley** [reply]
thank you
